inference-engine/src/mkldnn_plugin/nodes/common/tensor_desc_creator.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ie_layouts.h>
+
+namespace MKLDNNPlugin {
+
+enum class TensorDescCreatorTypes {
+    plain,
+    perChannel,
+    channelBlocked8,
+    channelBlocked16
+};
```

Comment:
We need to discuss unified names for common formats. I can propose alternative variant which is aligned with oneDNN naming convention and seems to be more clear and extensible: `ncsp`, `nspc`, `nCsp8c`, `nCsp16c`.
@openvinotoolkit/openvino-ie-cpu-developers  any comments?

Reply:
The names have been changed in 9ba8ee1.